python/src/trezorlib/btc.py

```diff
@@ -191,6 +191,7 @@ def sign_tx(
     details: messages.SignTx = None,
     prev_txes: Dict[bytes, messages.TransactionType] = None,
     preauthorized: bool = False,
+    decred_staking_ticket: bool = False,
```

Comment:
this is useless, the field is covered by `kwargs` handling

Reply:
https://github.com/trezor/trezor-firmware/pull/1249/commits/20aa862004cd9731524b43026be377223078237c